### PR TITLE
New package: xfce4-panel-profiles-1.0.13

### PR DESCRIPTION
**Pull request by BlindRepublic**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

Closes: #40306

### CODE REVIEW

**Comment by TrueTechie** [comment]
Closes #40306

**Comment by classabbyamp** [comment]
> Closes https://github.com/void-linux/void-packages/issues/40306

put this in the pr description and/or the commit message so it auto-closes

File: srcpkgs/xfce4-panel-profiles/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --python=python3"
+hostmakedepends="intltool tar"
+depends="xfce4-panel python3-gobject"
+short_desc="Simple application to manage Xfce panel layouts"
+maintainer="TrueTechie <TrueTechie@mailo.com>"
+license="GPL-3.0-or-later"
+homepage="https://docs.xfce.org/apps/xfce4-panel-profiles/start"
+distfiles="https://archive.xfce.org/src/apps/${pkgname}/${version%.*}/${pkgname}-${version}.tar.bz2"
```
Comment (classabbyamp) [inline srcpkgs/xfce4-panel-profiles/template:13]:
```suggestion
distfiles="https://archive.xfce.org/src/apps/xfce4-panel-profiles/${version%.*}/xfce4-panel-profiles-${version}.tar.bz2"
```

File: srcpkgs/xfce4-panel-profiles/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'xfce4-panel-profiles'
+pkgname=xfce4-panel-profiles
+version=1.0.13
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --python=python3"
+hostmakedepends="intltool tar"
```
Comment (classabbyamp) [inline srcpkgs/xfce4-panel-profiles/template:7]: why does this need tar?

Comment (TrueTechie) [reply srcpkgs/xfce4-panel-profiles/template:7]: The install procedure requires it for the layouts as far as I can tell. Without it on the host system, it fails:

```
...
cd data/layouts/cupertino; tar  -cvjf "../Cupertino.tar.bz2" *
/bin/sh: line 1: tar: command not found
make: *** [Makefile:36: layouts] Error 127
```

Comment (classabbyamp) [reply srcpkgs/xfce4-panel-profiles/template:7]: ok just wanted to check

**Comment by TrueTechie** [comment]
> Pull Requests become stale 90 days after last activity and are closed 14 days after that. If this pull request is still relevant bump it or assign it.

This is relevant

**Comment by paper42** [comment]
Merge commits are not allowed, please rebase this on top of master

File: srcpkgs/xfce4-panel-profiles/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'xfce4-panel-profiles'
+pkgname=xfce4-panel-profiles
+version=1.0.13
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --python=python3"
```
Comment (paper42) [inline srcpkgs/xfce4-panel-profiles/template:6]: would this work?
```suggestion
build_style=gnu-configure
configure_args="--python=python3"
```

**Pull request closed without merging**